Access denied logic (#731)
overblog/GraphQLBundle: docs/security/fields-access-control.md

If `config.fields.*.access` value is true field will be normally resolved but will be `null` otherwise.

> Act like access is`true` if not set.

Reviewer: ```suggestion
Acts like access is`true` if not set.
```

> If `config.fields.*.access` value is true field will be normally resolved but will be `null` otherwise.
> Act like access is`true` if not set.

Reviewer: ```suggestion
If `config.fields.*.access` value is true, field will be resolved normally, otherwise will be `null`.
Acts like `access` is`true` if not set.
```

Additional configuration can be set using `config.fields.*.accessConfig` or globally with `config.fieldsDefaultAccessConfig`.  
The available option is :

> `nullOnDenied` : Boolean indicating if we should return `null` instead of raising an exception is the access to the field is denied. `default=false`.  **Warning: When using this option make sure the corresponding field is nullable.**   

Reviewer: ```suggestion
`nullOnDenied` : Boolean indicating if we should return `null` instead of raising an exception if the access to the field is denied. `default=false`.  **Warning: When using this option make sure the corresponding field is nullable.**   
```
